src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+  <div class="col-lg-4">
+    <fieldset>
+      <legend class="cd-header"
+              i18n>Cluster Resources</legend>
+      <table class="table table-striped">
+        <tr>
+          <td i18n
+              class="bold">Hosts</td>
+          <td>{{ hostsCount }}</td>
+        </tr>
+      </table>
+    </fieldset>
+  </div>
+
+  <div class="col-lg-8">
+    <legend i18n
+            class="cd-header">Hosts by Label</legend>
+    <cd-table [data]="hostsByLabel.data"
+              [columns]="hostsByLabel.columns">
+    </cd-table>
+
+    <legend i18n
+            class="cd-header">Host Details</legend>
+    <cd-table [data]="hostsDetails.data"
+              [columns]="hostsDetails.columns">
+    </cd-table>
+  </div>
+</div>
```

Comment:
While checking at this layout, I'm wondering if this is going to be easy to brand with PF. The [PF wizard](https://www.patternfly.org/v4/components/wizard/html) relies on `<main>` and `<footer>` components together with specific styles for `...main`, `...main-body`, `...footer`, etc.

@nizamial09 did you check that the current wizard is going to work with the PF one?

Reply:
Since for pf mostly we are doing the scss stylings the wizard and its stepper will work fine. And this is a component that we call inside the wizard. So this will work like the rest of the component where the branding styles are already being applied. but I am guessing some work will needs to be done in downstream. @epuertat